src/App.tsx

```diff
@@ -1,24 +1,238 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState, useEffect } from 'react';
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoNotification } from './components/TodoNotification';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Error } from './types/Error';
+import {
+  getTodos,
+  addTodos,
+  deleteTodos,
+  editTodos,
+} from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 10137;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[] | []>([]);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [error, setError] = useState<Error>(Error.None);
+  const [value, setValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [actionsTodosId, setActionsTodosId] = useState<number[] | []>([]);
+
+  const getTodosList = async () => {
+    try {
+      const todosFromData = await getTodos(USER_ID);
+
+      setTodos(todosFromData);
+    } catch {
+      setError(Error.Get);
+    }
+  };
```

Comment:
apply useCalback or put it inside the `useEffect `to avoid redeclaration on each render